libraries/chain/apply_context.cpp

```diff
@@ -184,10 +184,10 @@ void apply_context::exec_one()
       r.auth_sequence[auth.actor] = next_auth_sequence( auth.actor );
    }
 
-   trx_context.executed_action_receipt_digests.emplace_back( r.digest() );
-
    finalize_trace( trace, start );
 
+   trx_context.executed_action_receipts.compute_and_append_digests_from(trace);
```

Comment:
This should be moved back above `finalize_trace` otherwise the `action.elapsed` does not contain the time for creating the digest(s).

Reply:
Weird, I can't remember why I moved it! Moving it back.